samples/bluetooth/direct_test_mode/boards/nrf5340dk_nrf5340_cpunet.overlay

```diff
@@ -3,6 +3,11 @@
 	current-speed = <19200>;
 };
 
+&arduino_spi {
+	status = "disabled";
+	compatible = "nordic,nrf-spim";
+};
```

Comment:
In `fem/nrf21540.c` you are using specific SPI instance, so I think the following would more appropriate (no need to set the compatible property, since the point is to disable the node):
```
&spi0 {
	status = "disabled";
};
```

Reply:
Done for both samples